core/smart_contract_query.md

```diff
@@ -13,6 +13,8 @@ dto SmartContractQuery:
 
 ```
 dto SmartContractQueryResponse:
+    raw: any;
```

Comment:
`SmartContractQueryResponse` is a "core" class, but it's useful to hold the `raw` object here, as well (just like for objects returned by the network providers).